jujugui/static/gui/src/app/components/create-payment-user/create-payment-user.js

```diff
@@ -347,11 +348,17 @@ class CreatePaymentUser extends React.Component {
 CreatePaymentUser.propTypes = {
   acl: PropTypes.object.isRequired,
   addNotification: PropTypes.func.isRequired,
-  createCardElement: PropTypes.func.isRequired,
-  createToken: PropTypes.func.isRequired,
-  createUser: PropTypes.func.isRequired,
-  getCountries: PropTypes.func.isRequired,
   onUserCreated: PropTypes.func.isRequired,
+  payment: shapeup.shape({
+    createUser: PropTypes.func.isRequired,
+    getCountries: PropTypes.func.isRequired,
+    reshape: shapeup.reshapeFunc
```

Comment:
I don't see `reshape` being used anywhere. I also don't believe you need to pass it through.

Reply:
Thanks, I've cleaned these up and left only the ones that are required by `addReshape`.